.github/workflows/k8s-create-service-mesh.yml

```diff
@@ -0,0 +1,60 @@
+name: K8s Create Service Mesh
+
+on:
+  workflow_call:
+    inputs:
+      git-lfs:
+        description: "Checkout the project with git lfs"
+        type: boolean
+        default: false
+  workflow_dispatch:
+
+jobs:
+  k8s-setup-app:
+    runs-on: [self-hosted, docker]
+    if: contains('["acpoppe","KlausNie","kupeliorhun"]', github.triggering_actor)
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          lfs: ${{ inputs.git-lfs }}
+      - uses: azure/setup-kubectl@v3
+      - uses: azure/setup-helm@v3
+        with:
+          version: "3.13.3"
+      - uses: azure/k8s-set-context@v3
```

Comment:
NIT: do the context setting right before it is used the first time (installing linkerd)